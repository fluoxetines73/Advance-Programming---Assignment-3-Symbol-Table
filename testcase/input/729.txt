RPRINT
RPRINT
BEGIN
PRINT
RPRINT
RPRINT
BEGIN
INSERT a number
LOOKUP a
PRINT
RPRINT
END
RPRINT
END
INSERT b number
INSERT e_123_2r number
PRINT