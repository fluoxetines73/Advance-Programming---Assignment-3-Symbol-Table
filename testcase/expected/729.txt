




success
2
a//2
a//2

success
success
b//0 e_123_2r//0